specification/security/resource-manager/Microsoft.Security/preview/2023-02-01-preview/securityConnectors.json

```diff
@@ -0,0 +1,1494 @@
+          "description": "Is Microsoft Defender for Cloud Kubernetes agent auto provisioning enabled"
+        },
+        "policyAgentAutoProvisioningFlag": {
+          "type": "boolean",
+          "description": "Is Policy Kubernetes agent auto provisioning enabled"
+        }
+      }
+    },
+    "cspmMonitorGithubOffering": {
+      "type": "object",
+      "description": "The CSPM monitoring for github offering",
+      "x-ms-discriminator-value": "CspmMonitorGithub",
+      "allOf": [
+        {
+          "$ref": "#/definitions/cloudOffering"
+        }
+      ],
+      "properties": {}
+    },
+    "cspmMonitorAzureDevOpsOffering": {
+      "type": "object",
+      "description": "The CSPM monitoring for AzureDevOps offering",
+      "x-ms-discriminator-value": "CspmMonitorAzureDevOps",
+      "allOf": [
+        {
+          "$ref": "#/definitions/cloudOffering"
+        }
+      ],
+      "properties": {}
+    },
+    "defenderCspmAwsOffering": {
+      "type": "object",
+      "description": "The CSPM P1 for AWS offering",
+      "x-ms-discriminator-value": "DefenderCspmAws",
+      "allOf": [
+        {
+          "$ref": "#/definitions/cloudOffering"
+        }
+      ],
+      "properties": {
+        "vmScanners": {
+          "type": "object",
+          "description": "The Microsoft Defender for Server VM scanning configuration",
+          "properties": {
+            "enabled": {
+              "type": "boolean",
+              "description": "Is Microsoft Defender for Server VM scanning enabled"
+            },
+            "configuration": {
+              "type": "object",
+              "description": "configuration for Microsoft Defender for Server VM scanning",
+              "properties": {
+                "cloudRoleArn": {
+                  "type": "string",
+                  "description": "The cloud role ARN in AWS for this feature"
+                },
+                "scanningMode": {
+                  "type": "string",
+                  "description": "The scanning mode for the VM scan.",
+                  "enum": [
+                    "Default"
+                  ],
+                  "x-ms-enum": {
+                    "name": "scanningMode",
+                    "modelAsString": true
+                  }
+                },
+                "exclusionTags": {
+                  "type": "object",
+                  "additionalProperties": {
+                    "type": "string"
+                  },
+                  "description": "VM tags that indicates that VM should not be scanned"
+                }
+              }
+            }
+          }
+        },
+        "dataSensitivityDiscovery": {
+          "type": "object",
+          "description": "The Microsoft Defender Data Sensitivity discovery configuration",
+          "properties": {
+            "enabled": {
+              "type": "boolean",
+              "description": "Is Microsoft Defender Data Sensitivity discovery is enabled"
```

Comment:
>is [](http://example.com/codeflow?start=79&length=2)

nit, you use 'is' twice in this description, that looks wrong

Reply:
removed last is